packages/wonder-blocks-form/src/components/__tests__/checkbox-group.test.js

```diff
@@ -1,85 +1,138 @@
+        });
+
+        it("should set aria-invalid on choices when there's an error message", () => {
+            // Arrange, Act
+            render(<TestComponent errorMessage="there's an error" />);
+
+            const checkboxes = screen.getAllByRole("checkbox");
 
-    it("calls onChange for each new selection", () => {
-        // a is clicked
-        const a = group.find(Choice).at(0);
-        const aTarget = a.find("ClickableBehavior");
-        aTarget.simulate("click");
-        expect(onChange).toHaveBeenCalledTimes(1);
-
-        // now b is clicked, onChange should also be called
-        const b = group.find(Choice).at(1);
-        const bTarget = b.find("ClickableBehavior");
-        bTarget.simulate("click");
-        expect(onChange).toHaveBeenCalledTimes(2);
+            // Assert
+            expect(checkboxes[0]).toHaveAttribute("aria-invalid", "true");
+            expect(checkboxes[1]).toHaveAttribute("aria-invalid", "true");
+            expect(checkboxes[2]).toHaveAttribute("aria-invalid", "true");
+        });
+
+        it("checks that aria attributes have been added correctly", () => {
+            // Arrange, Act
+            render(<TestComponent />);
+
+            const checkboxes = screen.getAllByRole("checkbox");
+
+            // Assert
+            expect(checkboxes[0]).toHaveAttribute("aria-labelledby", "test-a");
+            expect(checkboxes[1]).toHaveAttribute("aria-labelledby", "test-b");
+            expect(checkboxes[2]).toHaveAttribute("aria-labelledby", "test-c");
+        });
     });
 
-    it("checks that aria attributes have been added correctly", () => {
-        const a = group.find(Choice).at(0);
-        const b = group.find(Choice).at(1);
-        const c = group.find(Choice).at(2);
-        expect(a.find("input").prop("aria-labelledby")).toEqual("test-a");
-        expect(b.find("input").prop("aria-labelledby")).toEqual("test-b");
-        expect(c.find("input").prop("aria-labelledby")).toEqual("test-c");
+    describe("flexible props", () => {
+        it("should render with a React.Node label", () => {
+            // Arrange, Act
+            const action = () =>
+                render(
+                    <CheckboxGroup
+                        label={
+                            <span>
+                                label with <strong>strong</strong> text
+                            </span>
+                        }
+                        groupName="test"
+                        onChange={() => {}}
+                        selectedValues={[]}
+                    >
+                        <Choice label="a" value="a" aria-labelledby="test-a" />
+                        <Choice label="b" value="b" aria-labelledby="test-b" />
+                        <Choice label="c" value="c" aria-labelledby="test-c" />
+                    </CheckboxGroup>,
+                );
+
+            // Assert
+            expect(action).not.toThrow();
```

Comment:
**suggestion:** Based on the test description, I think than rather than verifying that the component renders, we should probably check that the `label` is rendered (e.g. `screen.getByText(/label with.../i).toBeInTheDocument`). wdyt?